src/rpdk/core/plugin_registry.py

```diff
@@ -5,7 +5,13 @@
     for entry_point in pkg_resources.iter_entry_points("rpdk.v1.languages")
 }
 
-PLUGIN_CHOICES = sorted(PLUGIN_REGISTRY.keys())
+
+def get_plugin_choices():
+    plugin_choices = [
+        entry_point.name
+        for entry_point in pkg_resources.iter_entry_points("rpdk.v1.languages")
+    ]
```

Comment:
I would use set if you dont rely on element position

Reply:
Before my changes the list of plugins was already being sorted which converts the set back to a list. I f this is not required anymore I'll be ok with making it a set.